CRM/Upgrade/Incremental/MessageTemplates.php

```diff
@@ -112,6 +112,15 @@ protected function getTemplateUpdates() {
           ['name' => 'payment_or_refund_notification', 'type' => 'html'],
         ],
       ],
+      [
+        'version' => '5.14',
```

Comment:
version is a little tricky just because today is the day it changes - if changed right now it should be 5.15 - but later today the 5.15 rc will be cut & the master branch will be 5.16